spree_gateway.gemspec

```diff
@@ -30,6 +30,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'spree_core', '>= 3.7.0'
   s.add_dependency 'spree_extension'
+  s.add_dependency 'stripe'
```

Comment:
Interesting, so that's not already included?
I would suggest locking this dependency to a particular version - Gemfile.lock will be created in projects using this gem, and they may then resolve to an incompatible version.

Reply:
Yes, I added this to make manipulating the object coming back from Stripe via webhook easier. ActiveMerchant handles the rest of the Stripe <--> Spree logic, so was not previously included. Happy to lock it to the current version once we've got the other points 'locked down' (pun intended).